loadtest/results/cross-vm-affinity-probe-2026-08-28/affinity_probe.py

```
@@ -0,0 +1,147 @@
+
+
+def load_frame():
+    with open(FRAMES_PATH) as f:
+        return json.load(f)["frames"][0]
+
+
+def http(url, method="GET", body=None, headers=None, timeout=15):
+    data = json.dumps(body).encode() if body is not None else None
+    h = {"Content-Type": "application/json"}
+    h.update(headers or {})
+    req = urllib.request.Request(url, data=data, headers=h, method=method)
```

Ruff (0.16.2)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)